ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_general.schema.yml

```diff
@@ -58,3 +58,25 @@ keys:
                       name:
                         type: str
                         description: Dynamic Prefix List Name.
+      control_functions:
+        type: dict
+        description: Configure routing control function to control which routes to filter, update and accept from a peer or routing protocol.
```

Comment:
```suggestion
        description: |-
          Routing control functions (RCF) used to filter and update routes from a peer or during redistributions.
          Warning:
          This configuration cannot be pushed with `eos_config_deploy_eapi`, because of limitations in `arista.eos` and `ansible.netcommon` plugins.
          The configuration can be pushed via CloudVision with `eos_config_deploy_cvp` or `cv_deploy`.
```

Reply:
Updated.